gdctx/global.go

```diff
@@ -37,6 +38,7 @@ var (
 	OpVersion      int
 	EtcdProcessCtx *os.Process
 	EtcdClient     etcdclient.Client
+	EtcdInstance   *embed.Etcd
```

Comment:
I'm not sure if we need to keep the etcd instance in `gdctx`. Couldn't we just let it be in the `etcdmgmt` package itself?